src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +15,27 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        ArrayList<String> textFromFile = new ArrayList<>();
+        final char StartsWith = 'w';
+        try {
+            Scanner sc = new Scanner(file);
+            while (sc.hasNextLine()) {
+                String[] words = sc.nextLine().toLowerCase().split(" ");
+                for (int i = 0; i < words.length; i++) {
+                    if (words[i].startsWith(String.valueOf(StartsWith))) {
+                        textFromFile.add(words[i].replaceAll("[, '!.)?-]", ""));
+                    }
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Don’t ignore exceptions. Re-throw them as RuntimeException(e.g. throw new RuntimeException(e)) if you don’t expect your code to fail here, or make necessary actions at the catch block to fix the problem.
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_05.html

Reply:
fix this too